Code/Tools/AssetProcessor/native/ui/BuilderListModel.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <QAbstractListModel>
+#endif
+
+struct MyBuilderList : QAbstractListModel
```

Comment:
is there a reason this is called MyBuilderList instead of BuilderListModel, which would match the filename?